clang/test/CodeGenHIP/dynamic-alloca.cpp

```diff
@@ -0,0 +1,532 @@
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py UTC_ARGS: --version 5
+// RUN: %clang_cc1 -triple amdgcn-amd-amdhsa -x hip -emit-llvm -fcuda-is-device -o - %s | FileCheck %s
```

Comment:
I think there are already some clang tests for this, not sure what the point of this part is. We do need unit tests which execute the code

Reply:
are you referring to hip-tests here?

Reply:
> I think there are already some clang tests for this

Yes, there are some OpenCL tests similar to this. 

> not sure what the point of this part is.

I was looking to test these builtins for HIP. Is this needed?

> We do need unit tests which execute the code

We have plans of adding hip-tests, or do you want the code to be exercised in the llvm-test-suite

Reply:
I guess 51a014cb2d9c6f8303f9b11ffc035d69cbeb9e21 didn't add any hip tests, so I guess we need some.

> We have plans of adding hip-tests, or do you want the code to be exercised in the llvm-test-suite

llvm-test-suite would be better